src/InputFileResolver.ts

```diff
@@ -23,11 +22,46 @@ export default class InputFileResolver {
     return Promise.all([this.inputFileResolver.resolve(), this.mutateResolver.resolve()]).then(results => {
       const inputFiles = results[0];
       const mutateFiles = results[1];
-      markAdditionalFilesToMutate(inputFiles, mutateFiles.map(m => m.path));
-      logFilesToMutate(inputFiles);
+      InputFileResolver.markAdditionalFilesToMutate(inputFiles, mutateFiles.map(m => m.path));
+      InputFileResolver.logFilesToMutate(inputFiles);
       return inputFiles;
     });
   }
+
+  private static validateFileDescriptor(maybeInputFileDescriptors: Array<InputFileDescriptor | string>) {
```

Comment:
Do we still have to keep all of these methods static? They only seem to be called from inside non-static functions and they're private.